source/language/delays.rst

```diff
@@ -297,27 +295,27 @@ within a box are implementation details of the box; they are invisible
 to the outside scope and therefore do not prevent commutation.
 
 We introduce a ``boxas`` expression for labeling a box. We primarily use this to
-later refer to the length of this box. Boxed expressions are good for
+later refer to the duration of this box. Boxed expressions are good for
 this because their contents are isolated and cannot be combined with
 gates outside the box. Therefore, no matter how the contents of the box
-get optimized, the ``lengthof(boxlabel`` has a well-defined meaning.
+get optimized, the ``lengthof(boxlabel)`` has a well-defined meaning.
```

Comment:
Per @levbishop, `boxas` should also be dropped. He noted that it can be replaced with:
`box[<stretch_var>] {...}` and use `stretch_var` in lieu of `lengthof(<label>)`

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/f56961122910730ee52c6b091dc1cfedfc28986b.